Binaries.csv

```diff
@@ -1,16 +1,32 @@
 Directory,Filename,URL,sha256,wget options
 Animations,EONSLocustA.ukx,https://cloud.greatemerald.eu/s/G3ASKqJs02osmbG/download,c9f1cf6bc87c82a7c9b29cc4c74ebb4e41be6bf6c3bc767308e94f700118d32e,
+Animations,UT3FuryAnims.ukx,https://drive.google.com/uc?export=download&id=18BLUncfyfx3lwPzylUXwDt13XpzF8W-V,baf219a06c14d686a07d8983e7dc55892969ff064211307507e97173f2490d9f,--content-disposition
+Animations,UT3GoliathAnims.ukx,https://drive.google.com/uc?export=download&id=1GgJ7DrYwzKgvlijk5lPdhI1ulasfE_Bp,570b168bab316df8ca179cb0323fa490509ca9cf25c26dcfa078705395e02ff5,--content-disposition
+Animations,UT3MantaAnims.ukx,https://drive.google.com/uc?export=download&id=13YoAtFz4ftBbuo0IgDSYCO1PtwmH9ZbS,6add43660944f861d616f0bc6b01d1ef65ba7ac27fdc28178dc8e3006c78ec23,--content-disposition
 Animations,UT3NemesisAnims.ukx,https://cloud.greatemerald.eu/s/3hvScOUAFpuC89k/download,5717e6ddbe6336e2e3405ec08276c926bdc6ff12cdf608559a26d31a8bd60513,
 Animations,UT3NightshadeAnims.ukx,https://cloud.greatemerald.eu/s/5pAhIZsmcsmmXt7/download,876447b1d5f9bca5fc589981b88a5223b58e6e8f781c1607fb1aca945f5273bd,
+Animations,UT3PaladinAnims.ukx,https://drive.google.com/uc?export=download&id=1c5lknHX3QvrfAccpvxneiM6f_6jf2Ipv,b609d22f3de275c6f946bc42164e84670f60b0b214a327ffdffcfefbd8fbe9d8,--content-disposition
+Animations,UT3StealthbenderAnims.ukx,https://drive.google.com/uc?export=download&id=1Zi_iCWQuKAau2R3kfhPGz9lY5y-eZedZ,93f76efa9e1d4ba63f99f5eb9abc6043992eb3efeba1fc25a1cfc542f139b766,--content-disposition
+Animations,UT3ViperAnims.ukx,https://drive.google.com/uc?export=download&id=16N9Yx25K3qy_-SEERs_4AEy634sJtcL2,218a01b314dbf2d8dcd43a1a9b35cbe030adace209d2474de8798190e14ecb5a,--content-disposition
+Animations,UT3VH_Cicada_Anims.ukx,https://drive.google.com/uc?export=download&id=1DZCJnLy7IRMpJzHNN8loy-05YxL0lbm0,1bbe6bf21be961abb950b6f0a1bf4e3edbbf5998386ad26e656859e8b92e08cf,--content-disposition
+Animations,UT3VH_Hellbender_Anims.ukx,https://drive.google.com/uc?export=download&id=1UVMWr1TS6UR3CvhCeTNkxDUJLrUm0wFS,6b112be3e0a4a35a5353a99770396e9bbc1182f931534f02ab191db836cc89f9,--content-disposition
```

Comment:
@HellDragon-HK The checksum does not match here. Check what's going on here.

Reply:
Which two here?

Reply:
`UT3VH_Hellbender_Anims.ukx`. The comment is always about the line at the very bottom of the box.